models/linkages/R/read_restart.LINKAGES.R

```diff
@@ -56,7 +61,9 @@ read_restart.LINKAGES <- function(outdir, runid, stop.time, settings, var.names
     if ("AGB.pft" %in% var.names) {
       forecast[[1]] <- rep(NA,length(settings$pfts))
     }
-    
+    if ("Fcomp" %in% var.names) {
+      forecast[[1]] <- NA #rep(NA,length(settings$pfts)) #already has C  #* unit.conv
```

Comment:
why is this not `rep(NA,length(settings$pfts))` as AGB.pft case?

just asking because above, their dimensions look the same
```
names(forecast[[1]]) <- paste0('Fcomp.',pft.names)
names(forecast[[1]]) <- paste0('AGB.pft.',pft.names)
```

maybe it doesn't matter

Reply:
👍